auth.proto

```diff
@@ -37,11 +41,11 @@ message Token {
     string accessType = 3;
 }
 
-message IdentityReq {
+message AuthenticateReq {
```

Comment:
Let's rename to `AuthNReq`, `AuthNResp`. Do the same for `AuthZReq` and `AuthZResp`.